rampwf/utils/cli/testing.py

```diff
@@ -92,7 +93,8 @@ def main(submission, ramp_kit_dir, ramp_data_dir, data_label,
         ]
     else:
         submission = [submission]
-
+    if os.path.exists("utils"):
+        sys.path.append(os.path.join(ramp_kit_dir))
```

Comment:
you added this because of my [comment in the previous PR](https://github.com/paris-saclay-cds/ramp-workflow/pull/277#pullrequestreview-675022084) saying that it would seem more natural to be able to do `from utils import foo` than `import foo`. I wanted to add to my comment that this would make us add the `ramp_kit_dir` in `sys.path` and that this was maybe not what we wanted either. So maybe `import foo`, although less natural would be better.